src/components/modulecore/ToggleButton.vue

```diff
@@ -2,12 +2,11 @@
   <v-btn-toggle
     borderless
     dense
-    color="onprimary"
```

Comment:
Are you sure you want to remove this line ?

Here's what it looks with it : 

![image](https://github.com/wegue-oss/wegue/assets/57899415/53abb99b-7272-4115-8169-b5ff9392c09d)

Here's what it looks without :

![image](https://github.com/wegue-oss/wegue/assets/57899415/bbb83063-01de-4217-874d-e71a67981b41)

It's a matter of taste for sure but PR title says remove `non-effective` color props and this one is...

Reply:
Totally correct, I did not recognize that. Either I happened to have a setup with the same colors or my old eyes fooled me. Glad you caught that! I'll come up with a correction.